docs/how_to/install.rst

```diff
@@ -465,6 +470,17 @@ setting::
 
     Please make sure both the ``static`` and ``media`` subfolders exist in your
     project and are writable.
+    
```

Comment:
During the tutorial this is not required as runserver takes care of this. Also: the correct way to do this is ``manage.py collectstatic`` command